src/hotspot/os/linux/osContainer_linux.hpp

```diff
@@ -43,6 +43,9 @@ class OSContainer: AllStatic {
 
  public:
   static void init();
+  static void print_version_specific_info(outputStream* st);
+  static void print_container_helper(outputStream* st, jlong j, const char* metrics);
+
```

Comment:
hpp file needs forward declaration of outputStream now, or include ostream.hpp, otherwise you break files that include this header but not ostream.

Reply:
Hi Thomas, I added the inclusion of ostream.hpp .
Thanks, Matthias